### DISCUSSION TITLE
Trouble installing Pmetrics

### DISCUSSION THREAD

**Discussion by MariaLarrosa** [LAPKB/Pmetrics#203] [Q&A]
Hi there!

I am trying to install Pmetrics following the indications specify in the web https://www.lapk.org/Pmetrics_install.php
However, I am stuck at Step 3. "Launch Rstudio and install the devtools package"
First, I got this message "Rtools is required to build R packages but is not currently installed. Please download and install the appropriate version of Rtools before proceeding" 

So I did install Rtools, but now I am getting other messages such as 
- "Warning: unable to access index for repository https://cran.rstudio.com/src/contrib:cannot open URL 'https://cran.rstudio.com/src/contrib/PACKAGES'"
- "Error in readRDS(dest) : error reading from connection"

Do you know which can be the problem and how can I solve it?
All the best,

**Comment by mhovd** [comment]
Are you perhaps trying to install Pmetrics behind a firewall, e.g. on a hospital network? Your error indicates that R is not able to communicate with CRAN.
